scripts/intfutil

```diff
@@ -138,12 +138,28 @@ def appl_db_port_status_get(appl_db, intf_name, status_type):
     if status is None:
         return "N/A"
     if status_type == PORT_SPEED and status != "N/A":
-        status = '{}G'.format(status[:-3])
+        if (len(status) < len('1000')):
```

Comment:
I think it would be great if you can extract these lines to a new methods and reuse it
And it might be better to convert to numbers than manipulate text